lib/protocol/redis/methods/hashes.rb

```diff
@@ -43,13 +45,15 @@ def hset(key, field, value)
 				# @param key [Key]
 				# @param field [String]
 				# @param value [String]
+				# @return [Boolean]
```

Comment:
Maybe clarify "@returns [Boolean] Whether the key already existed".

Reply:
@ioquatix can do but I think it would be mostly copy/paste from redis docs. btw I think `@return` is expected keyword in rdoc/yard or there is the reason why it should me `@returns`?